.github/workflows/gradle-build-deploy.yml

```diff
@@ -30,20 +30,29 @@ jobs:
         name: website
         path: build/site
         
-
   deploy:
     needs:
-      - release
+    - release
     runs-on: ubuntu-latest
     if: github.event_name == 'push' && github.repository == 'jameshughes89/cs102'
-    steps:
-      - name: Configure AWS credentials from Production account
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: ${{ secrets.AWS_REGION }}
+    concurrency: deploying
+    steps: 
+    - uses: actions/checkout@v2
+    - uses: actions/download-artifact@v2
+      with:
+        name: website
+    - name: Unzip website
+      run: unzip website.zip
+    - name: Remove website.zip
+      run: rm website.zip    
```

Comment:
If it doesn't interfere with the rest of the job, I'd say leave the zip file there. Actions runners are ephemeral anyways, they'll clean it up.